server/aws-lsp-codewhisperer/src/language-server/utils.ts

```diff
@@ -92,5 +92,6 @@ export const flattenMetric = (obj: any, prefix = '') => {
 
 export function getLoginTypeFromProvider(credentialsProvider: CredentialsProvider): LoginType {
```

Comment:
Realising that naming for this method is bit confusing. We have 2 fields related to credentials logic: Creds type (IAM, SSO) and this login type for SSO (IdC, BuilderID). This is is only related to SSO, can we rename this to make it explicit? Something like `ssoConnectionType`?

Reply:
right, makes sense. Let me follow up with all the mentioned comments in a follow up PR to unblock Code Coverage one.